provider/resources.go

```diff
@@ -256,15 +256,15 @@ func Provider() tfbridge.ProviderInfo {
 							TypeSpec:    schema.TypeSpec{Type: "string"},
 						},
 						"repoDigest": {
-							Description: "The digest of the manifest pushed to the registry, e.g.: repo[:tag]@<algorithm>:<hash>",
+							Description: "The digest of the manifest pushed to the registry, e.g.: repository@<algorithm>:<hash>",
```

Comment:
Suggest adding:
* That for images pushed to a repo, this will be a unique image reference per build/push.
* That this should be used to pass to container orchestrators as the image reference to ensure they redeploy on changes to the pushed image. 
*  give a few concrete examples of what the output can look like
* Note that it is not available when the image is not pushed.